### PR TITLE
EBS root disks for all EC2 instances; drop worker secondary disk

### PR DESCRIPTION
**Pull request by bcwaldon**

### CODE REVIEW

File: roles/configure/templates/create-stack.sh.j2 [hunk 1]
```diff
@@ -15,11 +15,16 @@ PARAMETERS=(
...
 	"ParameterKey=GatewayInstanceType,ParameterValue={{ gateway_instance_type }}"
 	"ParameterKey=GatewayInstanceCount,ParameterValue={{ gateway_instance_count }}"
+	"ParameterKey=GatewayInstanceRootDiskSize,ParameterValue={{ gateway_instance_root_disk_size_gb }}"
 	"ParameterKey=ControllerInstanceType,ParameterValue={{ controller_instance_type }}"
 	"ParameterKey=ControllerInstanceCount,ParameterValue={{ controller_instance_count }}"
+	"ParameterKey=ControllerInstanceRootDiskSize,ParameterValue={{ controller_instance_root_disk_size_gb }}"
 	"ParameterKey=WorkerInstanceType,ParameterValue={{ worker_instance_type }}"
 	"ParameterKey=WorkerInstanceCount,ParameterValue={{ worker_instance_count }}"
+	"ParameterKey=WorkerInstanceRootDiskSize,ParameterValue={{ worker_instance_root_disk_size_gb }}"
+	"ParameterKey=WorkerInstanceRootDiskIOPs,ParameterValue={{ worker_instance_root_disk_size_gb * 30 }}"
```
Comment (luciferous) [inline roles/configure/templates/create-stack.sh.j2:25]: Should this be a "hidden" key since we always want to derive IOPs from size?

**Comment by luciferous** [comment]
LGTM 👍

File: roles/configure/templates/create-stack.sh.j2 (hunk 1 above)
Comment (bcwaldon) [inline roles/configure/templates/create-stack.sh.j2:25]: @luciferous what do you mean by a "hidden" key?

Comment (luciferous) [inline roles/configure/templates/create-stack.sh.j2:25]: Maybe prefix it with an underscore?

Comment (luciferous) [inline roles/configure/templates/create-stack.sh.j2:25]: Up to you, I don't feel strongly.

Comment (bcwaldon) [inline roles/configure/templates/create-stack.sh.j2:25]: I don't think it's necessary. It's not easy (or even possible?) to calculate it in CloudFormation, so from the template's perspective it's a required value. I'm not going to establish a new class of vars just yet.

**Pull request merged**